src/gen_experiments/config.py

```diff
@@ -41,77 +49,43 @@ def addn(x):
 
 
 plot_prefs = {
-    "test": _PlotPrefs(True, False, ({"sim_params.t_end": 10},)),
+    "test": _PlotPrefs(),
     "test-absrel": _PlotPrefs(
-        True, _convert_abs_rel_noise, ({"sim_params.noise_abs": 1},)
+        True, _convert_abs_rel_noise, GridLocator(..., {("sim_params.noise_abs", (1,))})
     ),
     "test-absrel2": _PlotPrefs(
         True,
         _convert_abs_rel_noise,
-        (
-            {"sim_params.noise_abs": 0.1},
-            {"sim_params.noise_abs": 0.5},
-            {"sim_params.noise_abs": 1},
-            {"sim_params.noise_abs": 2},
-            {"sim_params.noise_abs": 4},
-            {"sim_params.noise_abs": 8},
+        GridLocator(
+            ...,
+            (..., ...),
```

Comment:
What does `...` signify here in the GridLocator arguments? Is it meant to be taking all the metrics or is it empty?

Reply:
Nice!  Yeah, I wanted to borrow some notation from numpy that means "all elements".  They're both what are called "singletons".  Only one `NoneType` object exists in a python program, just like only one `EllipsisType` object ever exists.

I suppose I could have used `None` instead.  That may be an arguably better way, I just wanted to use an object that people associate with "all" of something.